### ISSUE TITLE
Opening Ipython link

### ISSUE THREAD

**Issue by Empire1126** [spyder-ide/spyder#10102]
### What steps will reproduce the problem?

clicking the IPython link in the section "Recommended first steps for Python beginners" caused this error to occur. However my browser did open the page correctly while still throwing this error. 

### Traceback
```python-traceback
  File "F:\Anaconda\envs\EECS 4750\lib\site-packages\spyder\widgets\browser.py", line 168, in createWindow
    webbrowser.open(to_text_string(self.url().toString()))
  File "F:\Anaconda\envs\EECS 4750\lib\webbrowser.py", line 78, in open
    if browser.open(url, new, autoraise):
  File "F:\Anaconda\envs\EECS 4750\lib\webbrowser.py", line 581, in open
    os.startfile(url)
ValueError: startfile: filepath too long for Windows
```

## Versions

* Spyder version: 3.3.6 
* Python version: 3.7.4
* Qt version: 5.9.6
* PyQt5 version: 5.9.2
* Operating System: Windows 10

### Dependencies

```
pyflakes >=0.6.0  :  2.1.1 (OK)
pycodestyle >=2.3 :  2.5.0 (OK)
pygments >=2.0    :  2.4.2 (OK)
sphinx >=0.6.6    :  2.1.2 (OK)
rope >=0.9.4      :  0.14.0 (OK)
jedi >=0.9.0      :  0.15.1 (OK)
nbconvert >=4.0   :  5.5.0 (OK)
pandas >=0.13.1   :  None (NOK)
numpy >=1.7       :  None (NOK)
sympy >=0.7.3     :  None (NOK)
cython >=0.21     :  None (NOK)
qtconsole >=4.2.0 :  4.5.4 (OK)
IPython >=4.0     :  7.7.0 (OK)
matplotlib >=2.0.0:  None (NOK)
pylint >=0.25     :  2.3.1 (OK)
```

**Comment by ccordoba12** [comment]
Thanks for reporting. We'll take a look at this in a future release.

**Comment by jitseniesen** [comment]
This is the same as issue #9849 which is fixed in PR #9857, so I'm closing this report. The fix is included in the last beta version (4.0.0beta4) and will also be in the version 4.0 release.
